summary.md

```diff
@@ -0,0 +1,23 @@
+# Summary
+
+The Xing Framework is a combined web development environment for building both a server backend (in [Ruby on Rails](http://rubyonrails.org/)) and a single-page browser frontend (in [AngularJS](https://angularjs.org/)). 
+
+The Xing Framework provides integration and coordination of these two frameworks.  Specifically:
+
+## Features: Why you should use Xing for your next web application
+* **Development Support**. Tooling for building and launching the entire stack in development. With a single command, you can compile and boot your whole frontend and backend stack at once and open your app in a browser.
```

Comment:
More superlative... it's not just a couple tools... it's an awesome integrated suite for launching and working on your SPA